web_src/js/utils/dom.js

```diff
@@ -194,3 +194,20 @@ export function loadElem(el, src) {
     el.src = src;
   });
 }
+
+// some browsers like PaleMoon don't have "SubmitEvent" support, so we need to polyfill it (a tricky method): use the last clicked button as submitter
+const needSubmitEventSubmitterPolyfill = typeof SubmitEvent === 'undefined';
+
+export function submitEventSubmitterGet(e) {
+  return needSubmitEventSubmitterPolyfill ? (e.target._submitter || null) : e.submitter;
+}
+
+export function initSubmitEventSubmitterPolyfill() {
+  if (!needSubmitEventSubmitterPolyfill) return;
+  console.warn(`This browser doesn't have "SubmitEvent" support, use a tricky method to polyfill`);
+  document.addEventListener('mousedown', (e) => {
```

Comment:
down? Not up?

Reply:
IIRC most orders are mousedown -> click -> action -> mouseup

Reply:
Ah, you mean that the normal action would otherwise be executed?
Hmm… Often enough I find myself pressing on a button and mid-press moving away from it to cancel it.
With `down`, this won't be possible.